packages/services/schema/src/lib/__tests__/federation-tag-extraction.spec.ts

```diff
@@ -370,6 +370,46 @@ describe('applyTagFilterToInaccessibleTransformOnSubgraphSchema', () => {
       }
     `);
     });
+
+    test('mutation object type is returned in "typesWithAllFieldsInaccessible" if all its fields are excluded', () => {
+      const filter: Federation2SubgraphDocumentNodeByTagsFilter = {
+        include: new Set(),
+        exclude: new Set(['exclude']),
+      };
+      const sdl = parse(/* GraphQL */ `
+        schema @link(url: "https://specs.apollo.dev/federation/v2.0", import: ["@tag"]) {
+          query: Query
+        }
```

Comment:
Isn't `mutation: Mutation` missing here? By explicitly declaring `schema` and omitting `mutation` we are indicating that this subgraph has no Mutation fields and `Mutation` is just a "normal" object type without any special semantic meaning. 🤔

Reply:
Yeah it was... 😨